apis/k8ssandra/v1alpha1/k8ssandracluster_webhook.go

```diff
@@ -146,6 +146,10 @@ func (r *K8ssandraCluster) ValidateUpdate(old runtime.Object) error {
 		return ErrClusterName
 	}
 
+	if err := validateNoRackRenamed(oldCluster.Spec.Cassandra, r.Spec.Cassandra); err != nil {
+		return err
+	}
```

Comment:
I think this covers this from a few lines below:
```
// TODO Racks can only be added and only at the end of the list - no other operation is allowed to racks
```
Unless we want to control more fields in the rack configuration.